01-intro.Rmd

```diff
@@ -72,6 +72,9 @@ The following outline several key questions:
 
 To determine if your research requires compliance with other policies that may influence how you share your data, take this [quiz](https://sharing.nih.gov/other-sharing-policies/which-policies-apply-to-my-research).
 
+<div class = "ethics"> 
+In addition to these questions, there are ethical considerations that you may want to think about. See the [ethics section](https://hutchdatascience.org/NIH_Data_Sharing/ethics.html) of the appendix for more information. 
```

Comment:
Any reason not create a link e.g. {#ethics} ?

Reply:
just keeping it more separate kinda - but we can definitely do a link